### DISCUSSION TITLE
Is atomWithQuery dependant on Suspense?

### DISCUSSION THREAD

**Discussion by marcselman** [pmndrs/jotai#613] [Q&A] [answered]
I'm using React version 16.8.6. When I'm trying to use `atomWithQuery` I get an error 'ReactDOMServer does not yet support Suspense.'.
I could not find anything in the documentation about a dependancy on a minimal React version.

**Comment by dai-shi** [comment, marked as answer]
Async atoms require `Suspense` (for lazy loading, experimental for data) and minimal requirement is `react` 16.8 (no dependency on `react-dom`.) However, ReactDOMServer doesn't support `Suspense`, so it needs some techniques to avoid Suspense with async atoms in SSR. (It's like providing initial data, but documentation for SSR is still TODO.)

`atomWithQuery` is originally designed with async atoms, but afterwards @aulneau improved it. Now, with `initialData`, it doesn't depend on async atoms. (and it's not yet documented.)

So, your simple question reveals some doc issues. Thanks for it.
